daq_macros.py

```diff
@@ -2996,7 +2984,7 @@ def zebraCamDaq(angle_start,scanWidth,imgWidth,exposurePeriodPerImage,filePrefix
   setPvDesc("vectorNumFrames",numImages)    
   setPvDesc("vectorframeExptime",exposurePeriodPerImage*1000.0)
   setPvDesc("vectorHold",0)
-  zebraDaqPrep()
+  RE(zebraDaqPrep())
```

Comment:
I have addressed the methods for PvDesc in f9aaf01 - beamline_support now uses EpicsSignal devices to communicate with all of its PVs instead of pyepics PVs.

Reply:
since the LSDC server is completely separated from direct user control of its scans, there is the question of exactly where to call RE. I agree that the current method is not great, so I will propose that this be moved further up the call chain (still functions though). The question would be should it be somewhere like in runDCQueue, which is directly handling the request that comes into the queue, or should it be further down, such as collectData where it fans out to handle various scans differently?

The eScan currently has an embedded RE call within it, which is the next level down from collectData.

Reply:
> I have addressed the methods for PvDesc in [f9aaf01](https://github.com/NSLS-II/lsdc/commit/f9aaf016fc75b775918d8127ada2f357e4581ea1) - beamline_support now uses EpicsSignal devices to communicate with all of its PVs instead of pyepics PVs.

actually, I think this has a lot of consequences on the server side - the way the server handles many incoming commands will have to change, and all of those will have to run through RE. this requires more thought

Reply:
reverted the changes to beamline_support - this should not be a global thing. objects will be implemented with Ophyd as necessary.

at Wednesday's meeting, we agreed that we will implement one function (zebraDaq) fully with Ophyd and test for any unforeseen consequences.

Reply:
I have removed the RE calls and put one into collect_detector_seq_hw, which is along the code path for the standard experiment.